classification/network_params.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2015, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+from settings import NUM_CATEGORIES
+
+# Regions verbosity
+VERBOSITY = 0
+
+# Region names for the network
+SENSOR_REGION_NAME = "sensorRegion"
+SP_REGION_NAME = "spRegion"
+TM_REGION_NAME = "tmRegion"
+UP_REGION_NAME = "upRegion"
+CLASSIFIER_REGION_NAME = "classifierRegion"
+
+# Sensor types
+SENSOR_TYPE = "py.RecordSensor"
+
+# Encoder names
+SCALAR_ENCODER_NAME = "scalarEncoder"
+SCALAR_ENCODER_FIELD_NAME = "y"
+
+# Classifier types
```

Comment:
This is confusing... why not have a `CLASSIFIER_TYPE` the user defines as either "py.CLAClassifierRegion" or "py.KNNClassifierRegion"?